test/examples/src/ts/GetInfoFromExistingDatabase.ts

```diff
@@ -14,45 +14,37 @@
  * limitations under the License.
```

Comment:
Could you set a special 80 char line length for all of the examples? It was initially formatted to become more readable in the README.

Reply:
https://eslint.org/docs/rules/max-len tells me the default is 80 and I'm not seeing an override for that. Could you please clarify?

Reply:
I was asking this because previously in the examples, many lines were broken into two lines because those would have been under 80 chars/line, but now they do not. E.g.
```diff
-         console.log(`Document count in "${dbNameResult}" database is ` +
-            documentCount + ".");
+   console.log(`Document count in "${dbNameResult}" database is ` + documentCount + '.');
```

Reply:
Like aaa391c ?

Reply:
For the `*.js` files yes, but it did not change the `*.ts` files, unfortunately.

Reply:
I'll look into that.

Reply:
I think 85ea6ee made some changes there. Is it good now?